packages/jsii-pacmak/test/generated-code/__snapshots__/target-go.test.ts.snap

```diff
@@ -4,12 +4,25 @@ exports[`Generated code for "@scope/jsii-calc-base": <outDir>/ 1`] = `
 <root>
  ┗━ 📁 go
     ┗━ 📁 scopejsiicalcbase
+       ┣━ 📄 go.mod
        ┣━ 📁 jsii
        ┃  ┣━ 📄 jsii.go
        ┃  ┗━ 📄 tarball.embedded.go
        ┗━ 📄 scopejsiicalcbase.go
 `;
 
+exports[`Generated code for "@scope/jsii-calc-base": <outDir>/go/scopejsiicalcbase/go.mod 1`] = `
+module github.com/aws-cdk/jsii/jsii-calc/go/scopejsiicalcbase
+
+go 1.15
+
+require (
+	_jsii_ "github.com/aws-cdk/jsii/jsii-experimental"
```

Comment:
What is the module alias here? I haven't seen it in go.mod before.

Jsii runtime should also be versioned to the jsiiVerion provided in the assembly.